SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -31,37 +31,37 @@ SoD_SPLanguage = eng.json
 # Цвет координат курсора мыши на Карте Приключений
 # Die Koordinaten des Mauszeigers während man auf der Abenteuerkarte ist
 # Συντεταγμένες του κέρσορα του ποντικιού όταν βρίσκεστε στον χάρτη περιπέτειας (Adventure Map)
-AdventureCoord = H3Cyan
+AdventureCoord = Regular
 
 # The color of hit points of the creature under the mouse cursor while in combat
 # Цвет здоровья стека существ при наведении курсора мыши во время боя
 # Die Farbe der Lebenspunkte der Kreatur unter dem Mauszeiger während der Kampf
 # Το χρώμα των πόντων ζωής του πλάσματος κάτω από τον κέρσορα του ποντικιού κατά τη διάρκεια της μάχης
-HitPoints = H3Orange
+HitPoints = Regular
 
 # The color used when all targetted creatures will perish / or be resurrected
 # Цвет, означающий, что все целевого стека существ погибнут или будут воскрешены
 # Die benutzte Farbe wann alle gezielten Kreaturen sterben werden / oder sie auferstanden werden werden
 # Το χρώμα που χρησιμοποιείται όταν όλα τα στοχευμένα πλάσματα θα πεθάνουν / ή θα αναστηθούν# Το χρώμα που χρησιμοποιείται όταν μόνο ένα τμήμα των στοχευμένων πλασμάτων θα πεθάνει / ή θα αναστηθεί
-KillAll = H3Green
+KillAll = Regular
 
 # The color used when only a portion of targetted creatures will perish / or be resurrected
 # Цвет, означающий, что только часть целевого стека существ погибнет или будет воскрешена
 # Die benutzte Farbe wann nur ein Teil der gezielten Kreaturen sterben wird / oder ein Teil auferstanden werden wird
 # Το χρώμα που χρησιμοποιείται όταν μόνο ένα τμήμα των στοχευμένων πλασμάτων θα πεθάνει / ή θα αναστηθεί
-KillSome = H3Red
+KillSome = Regular
 
 # The color of move sequences while in combat
 # Цвет показателя очередности хода стеков в бою
 # Die Farbe der Bewegungsreihenfolge während der Kampf
 # Το χρώμα της ακολουθίας κινήσεων κατά τη διάρκεια της μάχης
-MoveSequence = H3Yellow
+MoveSequence = Regular
 
 # The color of coordinates under the mouse cursor while in combat
 # Цвет координат под курсором мыши во время боя
 # Die Farbe der Koordinaten unter dem Mauszeiger während der Kampf
 # Το χρώμα των συντεταγμένων κάτω από τον κέρσορα κατά τη διάρκεια της μάχης
-BattleCoord = Black
+BattleCoord = Regular
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.